kge/job/train.py

```diff
@@ -143,6 +143,8 @@ def _run(self) -> None:
             raise Exception(
                 f"{self.__class__.__name__} was initialized for forward only. You can only call run_epoch()"
             )
+        if self.epoch == 0:
+            self.save(self.config.checkpoint_file("init"))
```

Comment:
Why not ```0``` instead of ```"init"```?

Also, this checkpoint file is not deleted, right? (That's good, I think.)

Reply:
Naming it 0 can lead to it being deleted.
Also this naming avoids confusion whether our epoch counter starts at 0 or 1. This way it is clear, that nothing was trained on this checkpoint